state-chain/pallets/cf-ingress-egress/src/mocks.rs

```diff
@@ -0,0 +1,40 @@
+use cf_primitives::AffiliateShortId;
+use frame_support::parameter_types;
+use sp_core::ConstU32;
+use sp_runtime::BoundedBTreeMap;
+
+pub mod mock_btc;
+pub mod mock_eth;
+
+parameter_types! {
+	pub storage AffiliateMapping:  BoundedBTreeMap<(u64, AffiliateShortId), u64, ConstU32<100>> = Default::default();
+}
+
+pub struct MockAffiliateRegistry {}
```

Comment:
I intentionally didn't put in in cf_traits (see my note in the description to this PR). I prefer limiting code's visibility to only where it is needed. We can move it later if actually needed.

> We're likely to need it in other places (for example when it comes to processing payouts).

We will have full ids at that point.

Reply:
The convention of implementing the mock alongside the trait exists for good reason. We can implement it once, and it can serve as a 'reference implementation' of how the trait should behave. We know where to find it and we avoid duplicate implementations.

The 'only implement it where it's needed' approach works for you right now (and probably also for you in the future) because you know where it's used. If someone *does* want to use it later (and we can't anticipate every possible use case of a trait), they may not know that there is already a mock defined in the ingress-egress pallet, and decide to 'only implement it where it's needed'. Now we have two different mock implementations.

Reply:
>  they may not know that there is already a mock defined in the ingress-egress pallet, and decide to 'only implement it where it's needed'.

To make this assessment I would (reasonably) expect them to see how/where it is used 🤷‍♂️. Can't say I'm convinced, but don't have a strong opinion. I will move the mock like you suggest.

Reply:
Thank you.